src/rules/getter_return.rs

```diff
@@ -97,35 +114,33 @@ impl<'c> GetterReturnVisitor<'c> {
     for (span, msg) in &self.errors {
       self.context.add_diagnostic_with_hint(
         *span,
-        "getter-return",
+        CODE,
         msg,
-        "Return a value from the getter function",
+        GetterReturnHint::Return,
       );
     }
   }
 
   fn report_expected(&mut self, span: Span) {
     self.errors.insert(
       span,
-      format!(
-        "Expected to return a value in '{}'.",
+      GetterReturnMessage::Expected(
         self
           .getter_name
           .clone()
-          .expect("the name of getter is not set")
+          .expect("the name of getter is not set"),
```

Comment:
We could potentially write this one as:
```rust
variant!(GetterReturnMessage, Expected, self.getter_name.unwrap())
```

Reply:
Currently `variant!` macro is in `test_util.rs`, which is compiled only when in tests due to `#[cfg(test)]`. 
Do you think `variant!` should be able to be used outside tests?

Reply:
Let's leave it in `test_util.rs` for a bit and see if it will be useful in other rules 👍

Reply:
OK I see.